Add es-ES translation

joesfer opened the pull request:
This pull request includes translations from English to Spanish (Spain).

joesfer commented on locales/es-ES.yml:
```diff
@@ -0,0 +1,216 @@
+  save: Guardar
+  update: Actualizar
+  view: Ver
+  welcome: ¡Hola!
```
Spanish is a gendered language; most words have a he/she variant. A literal translation for welcome (Bienvenido) would be (he)Bienvenido or (she)Bienvenida. To avoid dated language, I've chosen a more liberal "Hi"

joesfer commented on locales/es-ES.yml:
```diff
@@ -0,0 +1,216 @@
+  email_address: Correo electrónico
+  first_name: Nombre
+  forgot_password: Olvidé mi contraseña # login form link
+  forgot_password_title: Otra vez # heading on /password/new (hint: https://www.google.com/search?q=here+we+go+again+meme)
```
I've done my best. Unfortunately, memes don't translate well :)

joesfer commented on locales/es-ES.yml:
```diff
@@ -0,0 +1,216 @@
+      connected: Conectado
+      new_and_popular: Nuevo y Popular
+    time_saved: # partial (widget)
+      title: Tiempo ahorrado
```
I've done my best guessing, though some terms might benefit from context for an accurate translation. For example, I'm taking "saved time" as in "you're saving time" and not "serialising or checkpointing some time value". Happy to make corrections as you see fit.

joesfer commented on locales/es-ES.yml:
```diff
@@ -0,0 +1,216 @@
+  last_name: Apellido
+  login: Iniciar sesión
+  log_out: Cerrar sesión
+  minimum_characters: Caracteres como mínimo # ex: 8 characters minimum
```
Some sentences change depending on how the fragment is used. For example:

"8 minimum characters" can translate as "Mínimo 8 caracteres" or "8 caracteres como mínimo". The correct translation would depend on how the string is pieced together.

joesfer commented on locales/es-ES.yml:
```diff
@@ -0,0 +1,216 @@
+      title: Plugins
+      cta: Ir a Plugins
+      connected: Conectado
+      new_and_popular: Nuevo y Popular
```
I've kept the original English capitalisation but similar phrases seem to vary slightly. Please let me know if there's any unwanted inconsistency.

ryanckulp commented:
thank you @joesfer for all your thoughtfulness here. would you mind adding ~15 more key/value pairs based on this latest commit?
https://github.com/usetrmnl/localizations/commit/e3922464eb99e4f67a6e37af672312d362229873

joesfer commented:
> thank you @joesfer for all your thoughtfulness here. would you mind adding ~15 more key/value pairs based on this latest commit? [e392246](https://github.com/usetrmnl/localizations/commit/e3922464eb99e4f67a6e37af672312d362229873)

This is done now! I've incorporated the additional strings and rebased the branch on top of the latest `main` commit.

ryanckulp commented:
you rock @joesfer ! thank you again.
